pkgs/development/libraries/duckdb/default.nix

```diff
@@ -97,10 +97,13 @@ stdenv.mkDerivation (finalAttrs: {
         "test/sql/attach/attach_remote.test"
         "test/sql/copy/csv/test_sniff_httpfs.test"
         "test/sql/httpfs/internal_issue_2490.test"
+        # fails with incorrect result
+        "test/sql/copy/file_size_bytes.test"
```

Comment:
Do we know why this fails with incorrect result? Is this an upstream issue?

Reply:
I do not know why it fails.

Reply:
I think it must be some flaky test. I tried to run it locally and it did pass for me. Do you have log output from when this failed?

Reply:
The hydra log from the initial issue shows the failure: https://hydra.nixos.org/build/274192646/nixlog/1